### PR TITLE
Readme updated with supplement for publication

### PR DESCRIPTION
**Pull request by comorbidity**
* README updated 
* README includes references to Chart Review  (LabelStudio documentation)
* README includes references to PSM (cumulus-library CORE capabilty) 
* Supplement files imported, specific to this study

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-# cumulus-library-suicidality-icd10
-Validation of ICD-10 code usage of Suicidality Cases Compared to Patient Chart Review
+# How reliable are ICD-10 diagnosis codes to identify cases of pediatric suicidality?
+
+This study repository includes:
+
+* **Suicidality ICD10 case definition** (SQL on FHIR) 
+* [Propensity Score Matching](https://github.com/smart-on-fhir/cumulus-library/blob/main/tests/test_psm.py) (PSM) to select "case matched comparators"
```
Comment (dogversioning) [inline README.md:6]: This probably shouldn't link to the unit test. Some other options:
- link to [the documentation](https://docs.smarthealthit.org/cumulus/library/statistics/propensity-score-matching.html), and note that the module was generated based on the work done for this study
- link to the [PSM module](https://github.com/smart-on-fhir/cumulus-library/blob/main/cumulus_library/builders/psm_builder.py), with the same comment above
- The supplement document

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+* [Chart Review](https://docs.smarthealthit.org/cumulus/chart-review/) software for reproducing results
```
Comment (dogversioning) [inline README.md:7]: this is not actually in the repo - perhaps move this to a seperate list of related software? or link to the supplement doc

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+
+For more information, [browse the Cumulus library documentation](https://docs.smarthealthit.org/cumulus/library).
+
+## Usage
+
+To install the module, run `git clone git@github.com:smart-on-fhir/cumulus-library-suicidality-icd10.git`.
+
+To install the module, run `pip install -e .`.
+
+This will add a `cumulus-library-suicidality-icd10` study target to `cumulus-library`.
```
Comment (dogversioning) [inline README.md:24]: if this is public - we could cut a package release and folks could install it directly from pypi?

Comment (comorbidity) [reply README.md:24]: Yes, since that doesn't exist yet, I put in local build instruction.

File: README.md (hunk 2 above)
Comment (comorbidity) [reply README.md:7]: This = Chart Review? Ok I'll make a related packages ....

**Review by dogversioning — APPROVED** [review]

**Pull request merged**